pkgs/development/python-modules/gstools/default.nix

```diff
@@ -0,0 +1,64 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+  setuptools-scm,
+  numpy,
+  cython,
+  extension-helpers,
+  hankel,
+  emcee,
+  meshio,
+  pyevtk,
+  scipy,
+  pytestCheckHook,
+}:
+
+buildPythonPackage rec {
+  pname = "gstools";
+  version = "1.5.2";
```

Comment:
version `1.6.0` is out since last week and there seems to be a hash mismatch now with the tag of version `1.5.2`